packages/react-instantsearch/src/components/SearchBox.js

```diff
@@ -120,6 +120,7 @@ class SearchBox extends Component {
   onSubmit = e => {
     e.preventDefault();
     e.stopPropagation();
+    this.input.blur();
```

Comment:
Are we sure we want to change permanently this behaviour, removing the focus each time someone submit a search when `searchAsYouType` is disabled? This can be used even if not on mobile, even if I'm not sure it makes sense. What do you think?